CMakeLists.txt

```diff
@@ -711,11 +693,11 @@ if(${PLAYER_AUDIO_BACKEND} MATCHES "^SDL2.*$")
 		else()
 			message(STATUS "MIDI playback: WildMidi")
 		endif()
-	elseif(${PLAYER_ENABLE_FMMIDI} MATCHES "ON")
+	elseif(PLAYER_ENABLE_FMMIDI)
```

Comment:
this will not work because the next check is for "Fallback" and "Fallback" is also true.
Maybe swap the elseifs